### DISCUSSION TITLE
local DNS name

### DISCUSSION THREAD

**Discussion by Zetto12** [Stirling-Tools/Stirling-PDF#718] [Q&A]
Hi all,
I have deployed Stirling via a docker-compose stack in Portainer. It works beautifully and I can access it via IP:Port I would like to give it a DNS name that can be resolved in the network (like stirling.local) can I do that from the settings.yml or the docker-compose file or do I need to install pihole or something to resolve the name?
Thank you, Martin

**Comment by xlandhenry** [comment]
If you want to access it on just a few devices you can edit hosts file on those individual devices but of course the best way is to set up your own DNS server to resolve the name.
